skore/tests/unit/sklearn/estimator/feature_importance/test_permutation_importance.py

```diff
@@ -13,6 +14,22 @@
 from skore import EstimatorReport
 
 
+@contextlib.contextmanager
+def check_changed(value):
```

Comment:
Let's making the context manager specialized to the `cache`: `check_cached_changed` (only about renaming).